jmix-flowui/flowui/src/main/java/io/jmix/flowui/UiProperties.java

```diff
@@ -59,6 +60,14 @@ public class UiProperties {
      */
     List<String> viewFileExtensions;
 
+    /**
+     * Sets the maximum time in seconds during which the file will be considered relevant.
+     * Makes sense for using the built-in PDF viewer in the Chrome browser.
+     *
+     * @see <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#response_directives">Cache-Control HTTP | MDN</a>
+     */
+    int fileDownloaderCacheMaxAge;
```

Comment:
If a variable stores a physical quantity, always mention the unit of change in the name (i.e. timeoutSec, timeoutMillis, distanceMiles)

See https://youtrack.haulmont.com/articles/PLAT-A-1433/Code-Best-Practices#Code-Structure